content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -7,7 +7,7 @@ description: Configure the kubelet's image credential provider plugin
 content_type: task
 ---
 
-{{< feature-state for_k8s_version="v1.24" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
 <!-- overview -->
```

Comment:
This is good to go. With caveats.

---

Ideally, open a follow up PR and in that PR:
- write the first occurrence of _exec plugins_ in italics
- consider rewording that italicized text to _credential helper plugins_
- tweak the text
  > a single subdomain segment, so `*.io` does NOT match `*.k8s.io`.

  and change it to use bold, not caps, for emphasis, like this
  ```markdown
  a single subdomain segment, so `*.io` does **not** match `*.k8s.io`.
  ```

---

Is this still the latest (alpha) version:
```
apiVersion: kubelet.config.k8s.io/v1alpha1
kind: CredentialProviderConfig
```
?

If not, we need a few updates.